docs/architecture/databaseUtil.md

```diff
@@ -4,6 +4,41 @@ The [DatabaseUtil](../src/utils/databaseUtil.ts) abstracts
 writing and reading to the file system,
 as well as parsing and serializing the game data.
 
+For now it is assumed that there is only one save file.
+
+## Delete
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant A as Actor
+  participant D as DatabaseUtil
+  participant F as File System
+
+  A ->> D: delete()
+  D ->> F: unlink()
+
+  note over D: Handle errors
```

Comment:
Updated in [this commit](https://github.com/normal-devs/klicker-knight/pull/23/commits/770344826cdc59a236ffe6fdf83fcf3ad41e01fb) to show error handling